antiplag/tests.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
```

Comment:
@OliverKanat should this test be placed here? I saw you created separate `/tests` directory.

Reply:
Moved it there.